app/decorators/decidim/participatory_process_decorator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+#
+# This decorator adds required associations between Decidim::ParticipatoryProcess and Departaments.
+#
+#
+require_dependency 'decidim/participatory_process'
+Decidim::ParticipatoryProcess.class_eval do
+  belongs_to  :areas,
```

Comment:
Why do we need this belongs_to as we already have area attribute @spuigmitjadxc ?